src/platforms/common/enriching-events/scopes.mdx

```diff
@@ -101,3 +101,44 @@ attach any context information at all by calling `clear` on the scope, while the
 
 Keep in mind that `with-scope` will not capture any exceptions that happen inside its callback function,
 and every error that happens there will be silently ignored and *not* reported.
+
+<PlatformSection supported={["java", "android"]}>
+
+## Kotlin Coroutines
```

Comment:
let's add a note that this is a beta version and may contain bugs, so people are aware of it, like Flutter

https://github.com/getsentry/sentry-docs/blame/master/src/includes/getting-started-primer/flutter.mdx#L3

Reply:
👍